rules/integrations/azure/persistence_azure_global_administrator_role_assigned.toml

```diff
@@ -52,24 +51,33 @@ Azure AD's Global Administrator role grants comprehensive access to manage Azure
 - Implement conditional access policies to restrict Global Administrator role assignments to specific, trusted locations or devices.
 - Review and update role assignment policies to ensure that only a limited number of trusted personnel have the ability to assign Global Administrator roles.
 - Enhance monitoring and alerting mechanisms to detect similar unauthorized role assignments in the future, ensuring timely response to potential threats.
-
-## Setup
-
-The Azure Fleet integration, Filebeat module, or similarly structured data is required to be compatible with this rule."""
+"""
 references = [
+    "https://securitylabs.datadoghq.com/articles/i-spy-escalating-to-entra-id-global-admin/",
     "https://docs.microsoft.com/en-us/azure/active-directory/roles/permissions-reference#global-administrator",
+    "https://www.microsoft.com/en-us/security/blog/2025/08/27/storm-0501s-evolving-techniques-lead-to-cloud-based-ransomware/"
 ]
-risk_score = 47
+risk_score = 73
 rule_id = "04c5a96f-19c5-44fd-9571-a0b033f9086f"
-severity = "medium"
-tags = ["Domain: Cloud", "Data Source: Azure", "Use Case: Identity and Access Audit", "Tactic: Persistence", "Resources: Investigation Guide"]
+severity = "high"
+tags = [
+    "Domain: Cloud",
+    "Domain: Identity",
+    "Data Source: Azure",
+    "Data Source: Microsoft Entra ID",
+    "Data Source: Microsoft Entra ID Audit Logs",
+    "Use Case: Identity and Access Audit",
+    "Tactic: Persistence",
+    "Resources: Investigation Guide"
+]
 timestamp_override = "event.ingested"
 type = "query"
 
 query = '''
-event.dataset:azure.auditlogs and azure.auditlogs.properties.category:RoleManagement and
-azure.auditlogs.operation_name:"Add member to role" and
-azure.auditlogs.properties.target_resources.0.modified_properties.1.new_value:"\"Global Administrator\""
+event.dataset:azure.auditlogs and
+    azure.auditlogs.properties.category:RoleManagement and
+    azure.auditlogs.operation_name:"Add member to role" and
+    azure.auditlogs.properties.target_resources.*.modified_properties.*.new_value: "\"Global Administrator\""
```

Comment:
Are there instances where this field:value combination occurs outside of `azure.auditlogs.properties.target_resources.0.modified_properties.1.new_value`? I don't see any in our stack or in telemetry cluster when applying this query. Also do you care that this rule captures `azure.auditlogs.properties.target_resources.0.type` for both `User` and `ServicePrincipal`? I don't know the specifics around the `ServicePrincipal` value but wanted to make sure you didn't want to limit this to capture the `User` value only.

Reply:
> Are there instances where this field:value combination occurs outside of azure.auditlogs.properties.target_resources.0.modified_properties.1.new_value

Not that I have seen from telemetry or our testing either. However, the integer assignments for field nodes are dynamic so I prefer to to capture the intended value if it appears elsewhere at the cost of performance.

> Also do you care that this rule captures azure.auditlogs.properties.target_resources.0.type for both User and ServicePrincipal?

Global Administrator is user-centric only whereas SPs typically have API permissions assigned to them. We do not expect any other value than `User` here.